vpr/src/route/route_tree_timing.cpp

```diff
@@ -660,7 +661,9 @@ bool verify_route_tree_recurr(t_rt_node* node, std::set<int>& seen_nodes) {
 void free_route_tree(t_rt_node* rt_node) {
     /* Puts the rt_nodes and edges in the tree rooted at rt_node back on the
```

Comment:
I'd move this comment to before the function declaration (our more common style now).